ogc-example/src/main/scala/geotrellis/server/ogc/conf/package.scala

```diff
@@ -73,20 +73,26 @@ package object conf {
    * HOCON doesn't naturally handle unquoted strings which contain decimals ('.') very well.
    *  As a result, some special configuration handling is required here to allow unquoted
    *  strings specifically when we know we're trying to decode a ColorMap.
+   * 
+   * @note It is currently difficult to handle double-keyed maps. A workaround
+   * has been provided, but it only works with doubles that explicitly decimal
+   * pad to tenths (0.0 is OK, 0 is to be avoided)
    */
   implicit def colormapReader: ConfigReader[ColorMap] =
     ConfigReader[Map[String, ConfigValue]].map { cmap =>
-      val numericMap = cmap.map({ case (k, v) =>
+      val numericMap = cmap.flatMap({ case (k, v) =>
```

Comment:
Btw a code style question, do we need `({` here?

Reply:
Also, flatMaping / maping over a map produces a map; so you don't need `.toMap` conversions